terraform/modules/provisioners/variables.tf

```diff
@@ -16,6 +16,17 @@ variable "num_vm_instances" {
   description = "The number of compute instances that are available"
 }
 
+variable "rms_private_endpoint_id" {
+  type        = string
+  description = "The OCID of the resource manager private endpoint"
+}
+
+variable "is_rms_private_endpoint_required" {
```

Comment:
NitPicking: Can we use "use_rms_endpoint" since default is true?

Reply:
ok, I will change this  variable.